library/pool.c

```diff
@@ -1150,6 +1152,16 @@ int dnet_io_init(struct dnet_node *n, struct dnet_config *cfg)
 		}
 	}
 
+	if (cfg->flags & DNET_CFG_JOIN_NETWORK) {
+		struct dnet_grpc_client_config client_cfg;
+		memset(&client_cfg, 0, sizeof(struct dnet_grpc_client_config));
+		client_cfg.thread_num = cfg->grpc_thread_num;
```

Comment:
Why `grpc_thread_num` not only in `dnet_grpc_*_config`? I saw that shaitan suggested to separate configs.

Reply:
Because it's a way to allow clients specify `grpc_thread_num` and it doesn't broke ABI.
It's a cause why we need a `grpc_thread_num` in `dnet_config` too.